### DISCUSSION TITLE
Why does code sandbox not support whitespaces in directory names?

### DISCUSSION THREAD

**Discussion by Mansehej** [codesandbox/codesandbox-client#6780] [Q&A]
I use a document site generator and the sidebar is autogenerated with the folder structure. However, codesandbox does not seem to support whitespaces in the directory names, which is causing issues in live edits.

What is the reason behind this?

**Comment by JamesACS** [comment]
Hey @Mansehej - 

We have a prettier integration that will automatically try to format your code to be cleaner on save, it shouldn't impact the running of the code though.
